sklearn/tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+'''
+The __init__.py file makes Python treat directories containing it as modules.
+Here it is required for nose tests to discover the directory, when run from outside the source directory.

```

Comment:
Could you refer @lesteve's [comment](https://github.com/scikit-learn/scikit-learn/issues/6849#issuecomment-223947884) instead pl?